views/signup.erb

```diff
@@ -0,0 +1,28 @@
+<link href="<%= url('/styles.css') %>" rel="stylesheet" type="text/css" />
+<h1> Sign up for Chitter </h1>
+<form action="/signup" method="POST">
+
+    <div>
+      <label> Name </label>
+      <input type="text" name= "name" />
```

Comment:
A nice way to ensure users don't submit any blank fields is to put 'required' at the end of these input lines, e.g. `<input type="text" name= "name" required>`
This will prevent them from submitting while the field is blank, raising a dynamic error message on the page, and can save you from having to write similar code into app.rb
Check here for more info: https://www.w3schools.com/tags/att_input_required.asp